static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -30,107 +50,183 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
         );
       });
   };
-  const ipv4Addresses = getIpAddresses("inet");
-  const ipv6Addresses = getIpAddresses("inet6");
+  const ip4Addresses = getIpAddresses("inet");
+  const ip6Addresses = getIpAddresses("inet6");
 
   return (
     <div className="p-panel p-instance-detail-panel">
       <div className="p-panel__header p-instance-detail-panel--header">
-        <div className="p-panel__title">Instance details</div>
-        <div className="p-panel__controls">
-          <Button
-            appearance="base"
-            className="u-no-margin--bottom"
-            hasIcon
-            onClick={onClose}
-          >
-            <i className="p-icon--close"></i>
-          </Button>
+        <div className="p-panel__title-wrapper">
+          <div className="p-panel__title">
+            <span className="u-no-margin--bottom">Instance summary</span>
+            <span className="u-no-margin--bottom u-truncate">
```

Comment:
Maybe add a title with content "view more", so when truncated you can see the text on hover.